quartz/plugins/emitters/cname.ts

```diff
@@ -10,20 +11,25 @@ export function extractDomainFromBaseUrl(baseUrl: string) {
 
 export const CNAME: QuartzEmitterPlugin = () => ({
   name: "CNAME",
-  async emit({ argv, cfg }) {
-    if (!cfg.configuration.baseUrl) {
+  async emit(ctx: BuildCtx) {
```

Comment:
```suggestion
  async emit(ctx) {
```

dont need a type here as its inferred